upgrade/class/Xoops/Upgrade/Smarty5ScannerOutput.php

```
@@ -0,0 +1,274 @@
+    /**
+     * Append one summary row.
+     *
+     * @param string $label    already-localised label
+     * @param int    $value    count
+     * @param string $cssClass optional row class (e.g. 'danger', 'warning', 'success')
+     *
+     * @return void
+     */
+    private function summaryRow(string $label, int $value, string $cssClass = ''): void
+    {
+        $tr = '' === $cssClass ? '<tr>' : '<tr class="' . $cssClass . '">';
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $tr. Configured minimum length is 3. (undefined)

(ShortVariable)